analyzer/src/funTest/assets/projects/synthetic/carthage-expected-output.yml

```diff
@@ -1,6 +1,6 @@
 ---
 project:
-  id: "Carthage:oss-review-toolkit:ort:"
+  id: "Carthage:oss-review-toolkit:ort:<REPLACE_REVISION>"
```

Comment:
You state in the commit message "The revision is the actual wanted information in this case." but you don't explain *why* that is the case. I would have thought we'd still want the resolved revision, so we don't get e.g. "master" as part of the project id, but the commit SHA1. Am I mistaken?

Reply:
From what I'm observing the `revision` is the commit SHA1. The `resolvedRevision` is even `null` while constructing the project:
```
    private fun getProjectInfoFromVcs(workingDir: File): ProjectInfo {
        val workingTree = VersionControlSystem.forDirectory(workingDir)
        val vcsInfo = workingTree?.getInfo() ?: VcsInfo.EMPTY
        val normalizedVcsUrl = normalizeVcsUrl(vcsInfo.url)
        val vcsHost = VcsHost.toVcsHost(URI(normalizedVcsUrl))

        return ProjectInfo(
            namespace = vcsHost?.getUserOrOrganization(normalizedVcsUrl),
            projectName = vcsHost?.getProject(normalizedVcsUrl) ?: workingDir.path.substringAfterLast("/"),
            revision = vcsInfo.revision
        )
    }
```

Reply:
Ah, right, the revision of a working tree is always resolved, and the resolved revision is null. Mind adding the "The resolvedRevision is even null" bit to the commit message as a brief explanation?

Reply:
Changed the commit message. Hope that makes the reasoning behind this change more clear.

Reply:
Commit message typos:
s/acutally/actually/
s/an version/a version/

Imperative:
s/using the revision/use the revision/

Also "resolved SHA1" of what :-) The commit! So let's say "resolved commit SHA1" :-)